frontend/src/styles/refreshCard.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
```

Comment:
Here the refresh card should have a font-size of 10px, and the space between the title and the refresh card should be set at 8px, looking at the Figma design, right now it looks too prominent on the page and would need to be reduced, thank you